<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>388</width>
    <height>315</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastrar</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lbl_nome">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>60</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>nome:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_nome">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>60</y>
      <width>211</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: rgb(237, 51, 59);
selection-color: rgb(28, 113, 216);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNoPredictiveText</set>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_valor">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>110</y>
      <width>50</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>valor:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_valor">
    <property name="geometry">
     <rect>
      <x>118</x>
      <y>108</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LeftToRight</enum>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">alternate-background-color: rgb(229, 36, 36);</string>
    </property>
    <property name="inputMethodHints">
     <set>Qt::ImhNone</set>
    </property>
    <property name="inputMask">
     <string>999.00</string>
    </property>
    <property name="maxLength">
     <number>6</number>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
    <property name="placeholderText">
     <string>digite o valor</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>220</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="autoFillBackground">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">selection-background-color: rgb(249, 240, 107);
font: 75 13pt &quot;Ubuntu&quot;;</string>
    </property>
    <property name="text">
     <string>Ok</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_fundo_Titulo">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>210</width>
      <height>30</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(12, 62, 124);</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Titulo_Cadastrar">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>11</y>
      <width>90</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; color:#faf02a;&quot;&gt;Cadastrar&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_data">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>160</y>
      <width>67</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Data:</string>
    </property>
   </widget>
   <widget class="QDateEdit" name="dateEdit">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>150</y>
      <width>110</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_dados">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>210</y>
      <width>89</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>visualizar</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>388</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
